### DISCUSSION TITLE
Menaged Metadata with PNP Webpart

### DISCUSSION THREAD

**Discussion by ps1601** [microsoft-search/pnp-modern-search#2360] [Ideas]
Hello All

I hope this would be an enhancement soon.

Looks like there a No way we can use PNP Search to Connect to Managed Metadata and Mimic Managed Metadata Navigation which is very Popular on ON PREMISE. There is no Modern Way to do it except using the PUBLISHING FEATURE on SPO.

Also the Current Refiner/Vertical can we extend it to filter for Management Metadata Refiners when a Search happens on the page.
@wobba @stevebeauge @mrjarbenne  @PooLP  @edarroudi

**Comment by stevebeauge** [comment]
Such feature would be awesome, but I foresee some troubles with URLs. Managed metadata navigation was a feature of classic publishing sites and allows to tweak the URLs (http://myserver/sites/somesite/term1/term2/term3) and so on.
I don't think it's possible using modern experience to "rewrite" url to a custom page.
Maybe the term could put within search (#) or as query parameter and be used in query template. the "only" missing part is the webpart that will build these tree

**Reply by FranckyC** [reply]
You're right @stevebeauge, a possible solution would be to use query string/hash parameters to mimic this behavior. Managed metadata navigation is a classic feature not available in the modern experience unfortunately.

For the taxonomy refiners, this is a known requirement but we didn't have time to implement it yet. See https://github.com/microsoft-search/pnp-modern-search/discussions/1366
